sdk/src/main/java/com/mapbox/maps/MapInitOptions.kt

```diff
@@ -11,9 +12,10 @@ import android.util.AttributeSet
  * MapView(Context, MapboxMapOptions). If you add a map using XML, then you can apply these options
  * using custom XML tags.
  *
- * @property context the context of the application.
+ * @property context The context of the MapView.
  * @property resourceOptions Resource options when using a MapView. Access token required when using a Mapbox service. Please see [https://www.mapbox.com/help/create-api-access-token/](https://www.mapbox.com/help/create-api-access-token/) to learn how to create one.More information in this guide [https://www.mapbox.com/help/first-steps-android-sdk/#access-tokens](https://www.mapbox.com/help/first-steps-android-sdk/#access-tokens).
  * @property mapOptions Describes the map options value when using a MapView.
+ * @property plugins The plugins that will be loaded as part of MapView initialisation
```

Comment:
probably need to document the Strings should be the full path of the plugin class?